test/integration/firewalls_test.go

```diff
@@ -19,7 +20,7 @@ var (
 func TestListFirewalls(t *testing.T) {
 	client, _, teardown, err := setupFirewall(t, []firewallModifier{
 		func(createOpts *linodego.FirewallCreateOptions) {
-			createOpts.Label = randString(12, lowerBytes, digits) + "-linodego-testing"
+			createOpts.Label = randString(12, lowerBytes, upperBytes) + "-linodego-testing"
```

Comment:
Firewall labels must begin with a letter, I guess the first run of this fixture got lucky, but subsequent runs threw a 400.